src/Time.cpp

```diff
@@ -88,6 +88,32 @@ timeval Time::toTimeval() const
     return tv;
 }
 
+std::vector<int> Time::toTimeValues() const
+{
+    int64_t microseconds = this->microseconds;
+
+    int days = microseconds / 86400000000;
+    microseconds -= days * 86400000000;
+    int hours = microseconds / 3600000000;
+    microseconds -= hours * 3600000000;
+    int minutes = microseconds / 60000000;
+    microseconds -= minutes * 60000000;
+    int seconds = microseconds / 1000000;
+    microseconds -= seconds * 1000000;
+    int milliseconds = microseconds / 1000;
+    microseconds -= milliseconds * 1000;
+
+    std::vector<int> timeValues;
```

Comment:
For (tiny) efficiency improvement: `timeValues.reserve(6);`
Alternatively, consider using `std::array<int,6>`, instead (if this library is allowed to use C++11).

Reply:
I don't think it is @saarnold ?

Reply:
As far as I know base-types does not contain C++11 code.
But I'm not sure about the current policy regarding C++11 in rock core libraries. Maybe @doudou can share some light here.